app/data/models/delinkedcalculation.js

```diff
@@ -0,0 +1,44 @@
+module.exports = (sequelize, DataTypes) => {
+  const number0 = 0
+  const number11 = 11
+  const number16 = 16
+  const number38 = 38
```

Comment:
I think if we can these variables would be nicer with more meaningful names - e.g. number38 -> maxSBILength. Just gives more context to the reader on what these actually mean